packages/forms/test/template_integration_spec.ts

```diff
@@ -1879,3 +2083,40 @@ class NgModelChangesForm {
 class NgModelChangeState {
   onNgModelChange = () => {};
 }
+
+@Component({
+  selector: 'ng-model-max',
+  template: `<form><input name="max" type="number" ngModel [max]="max"></form>`
+})
+class NgModelMaxValidator {
+  max!: number;
+}
+
+@Component({
+  selector: 'ng-model-min',
+  template: `<form><input name="min" type="number" ngModel [min]="min"></form>`
+})
+class NgModelMinValidator {
+  min!: number;
+}
+
+@Directive({selector: '[myDir]'})
+export class CustomDirective {
+  @Input() min!: number;
+  @Input() max!: number;
+  constructor() {}
+}
+@Component({
+  selector: 'ng-model-negative-min-max',
+  template: `
+  <form>
+    <input name="min" type="text" ngModel [min]="min" myDir>
+    <input name="min" type="text" ngModel [min]="max" myDir>
```

Comment:
Should it be like this?
 ```suggestion
    <input name="max" type="text" ngModel [max]="max" myDir>
```